actions/global_actions.jsx

```diff
@@ -405,23 +406,24 @@ export function loadDefaultLocale() {
 }
 
 let lastTimeTypingSent = 0;
-export function emitLocalUserTypingEvent(channelId, parentId) {
-    const t = Date.now();
-    const membersInChannel = ChannelStore.getStats(channelId).member_count;
-
-    const license = getLicense(getState());
-    const config = getConfig(getState());
-    if (license.IsLicensed === 'true' && config.ExperimentalTownSquareIsReadOnly === 'true') {
-        const channel = ChannelStore.getChannelById(channelId);
-        if (channel && ChannelStore.isDefault(channel)) {
-            return;
+export function emitLocalUserTypingEvent(channelId, parentPostId) {
+    const userTyping = async (actionDispatch, actionGetState) => {
```

Comment:
nit: why are these prefixed with `action`? Is there something action-specific about this version of `dispatch` and `getState`?

Reply:
Well, basically is to no "shade" the global dispatch and getState variables, avoiding this way the es-lint warning.

Reply:
Ah, that makes sense given the transitional nature of this file.